components/TaskCard/index.js

```diff
@@ -0,0 +1 @@
+export * from "./TaskCard";
```

Comment:
```suggestion
import { useState } from "react";
export const TaskCard = ({ project, index }) => {
  return (
    <li key={project.id} onClick={() => setVisible((prev) => !prev)}>
      {`P${index + 1}: `}
      {project.projectTitle}
    </li>
  );
};
```
You can put your code for the component into the `index` directly.
I changed the `++index` into `index + 1`, by the way. You don't want to increase the variable, you only want to display it increased by one.

Reply:
OK

Reply:
I removed the state since you're not using it. And I removed the key prop since it's already present when mapping over the array in `pages/index.js`. All that's left to do is accept the suggested changes and you're good!